.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
#dd - Can you suggest feedback on these changes?

Reply:
#dd - Can you suggest feedback on these changes?